js/app.js

```diff
@@ -533,6 +630,22 @@
 
 	// implement function here
 
+	const combineLists = (items1, items2) => {
+		items1.forEach( (curr1) => {
+			if( !curr1.hasOwnProperty('item') || !curr1.hasOwnProperty('price') ) {
+				throw new Error('List must contain item and price.');
+			}
+		});
+		items2.forEach( (curr2) => {
+			if( !curr2.hasOwnProperty('item') || !curr2.hasOwnProperty('price') ) {
+				throw new Error('List must contain item and price.');
+			}
+		});		
+		let items = items1.concat(items2);
+		return items;
+		//Test still failing.
```

Comment:
Sorry, this was actually an error in the test itself that was fixed after Taq sent out the homework. See [this diff](https://github.com/FEWDMaterials/JSR_Shopping_List_Functions/commit/82239677ab0decb3ae4a3e21eec58591bc81fc63#diff-41d794d24ff042b1f9ac211fc3f9f951R551).